src/layout/Popover.js

```diff
@@ -6,8 +6,8 @@ let { fullscreen } = styles
 
 // Simple popover
 export default observer(
-  ({ show, children, style }) =>
-    f.view(show) && (
+  ({ isOpen, children, style }) =>
+    f.view(isOpen) && (
```

Comment:
Let's make sure futil is used with the upper case `F` even in this file. Several lines more in the diff won't matter at this point.

Reply:
Good point, contexture-react was started before we adopted `F` over `f`. There are probably a bunch of places that do this in the repo